projects/js-packages/components/components/status/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { __ } from '@wordpress/i18n';
+import classNames from 'classnames';
+import styles from './style.module.scss';
+
+interface StatusProps {
+	status?: 'active' | 'error' | 'inactive' | 'action' | 'initializing';
+	label?: string;
+	className?: string;
+}
+
+const Status = ( { className, label, status = 'inactive', ...rest }: StatusProps ): JSX.Element => {
+	const defaultLabels: Record< string, string > = {
+		active: __( 'Active', 'jetpack' ),
+		error: __( 'Error', 'jetpack' ),
+		action: __( 'Action needed', 'jetpack' ),
+		inactive: __( 'Inactive', 'jetpack' ),
+		initializing: __( 'Setting up', 'jetpack' ),
+	};
+
+	return (
+		<span
+			className={ classNames(
+				styles.status,
+				{
+					[ styles[ `status--${ status }` ] ]: status,
```

Comment:
a minor suggestion:

```suggestion
					[ styles[ `is-${ status }` ] ]: status,
```

you should change the style rules, too.

Reply:
Updated to match the naming convention, thanks for identifying this! https://github.com/Automattic/jetpack/pull/30209/commits/23198d83bd62ea0051aad1d9c578157b73e60606